subworkflows/local/prelude/main.nf

```diff
@@ -14,9 +14,7 @@ workflow PRELUDE {
     output_file_markersheet
 
     main:
-    samplesheet.map{meta, image_tiles, dfp, ffp -> meta}.set{meta}
-
-    output_file_xml         = SUMMARY_XML(meta, xml).output
-    output_file_markersheet = SUMMARY_MARKERSHEET_LITERAL(meta, markersheet).output
-    output_file_samplesheet = SUMMARY_SAMPLESHEET(meta, samplesheet).output
+    output_file_xml         = ( xml.map{meta, xmlpath -> [meta, xmlpath]} | SUMMARY_XML ).output
+    output_file_markersheet = SUMMARY_MARKERSHEET_LITERAL( markersheet.map{[["id": "markers"], it]} ).output
+    output_file_samplesheet = (samplesheet.map{meta, image_tiles, dfp, ffp -> [meta, samplesheet] } | SUMMARY_SAMPLESHEET).output
```

Comment:
Even though it's unused in `SUMMARY_SAMPLESHEET`, we shouldn't be passing the channel itself as the second element in the tuple here. If anything, it might be a nested tuple of the remaining values.
```suggestion
    output_file_samplesheet = (samplesheet.map{meta, image_tiles, dfp, ffp -> [meta, [image_tiles, dfp, ffp]] } | SUMMARY_SAMPLESHEET).output
```